examples/branch_and_bound/main.py

```diff
@@ -0,0 +1,105 @@
+from explorateur import Explorateur, ExplorationType, SearchType
```

Comment:
can you increment the __version__.py to 1.3.0 so that I can push the latest version to PyPI and add a comment to changelog.txt for 1.3.0 with BnB please